### PR TITLE
Setup repo administrative files

### PR DESCRIPTION
**Pull request by vvalderrv**
Signed-off-by: Vanessa Rene Valderrama <vvalderrama@linuxfoundation.org>

### CODE REVIEW

**Review by vvalderrv — COMMENTED** [review]
This change is not ready to be merged. I'll be submitting another patch based on reviews.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,76 @@
...
+
+### Test Locally
+
+Check sanity by running the Jenkins job-builder script in this directory:
+
+    jenkins-jobs test -r jjb
+
+### Deploy the templates to the Jenkins sandbox
+
+Login (after requesting membership in group jquery-jenkins-sandbox-access) at the Jenkins sandbox:
```
Comment (tykeal) [inline README.md:45]: A link to https://docs.releng.linuxfoundation.org/en/latest/helpdesk.html should be put in here to help direct people to the support system for requesting sandbox access

File: README.md [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
+
+    [jenkins]
+    query_plugins_info=False
+    url=https://jenkins.openjsf.org/sandbox
+    user=YOUR-USER-NAME
+    password=YOUR-API-TOKEN
+
+Build and deploy a specific job using the EXACT job name.
+
+    jenkins-jobs --conf jenkins.ini update jjb your-job-name-here
```
Comment (tykeal) [inline README.md:70]: Change this to:

jenkins-jobs --conf jenkins.ini update -j jjb your-job-name-here

File: README.md [hunk 3]
```diff
@@ -1,4 +1,76 @@
...
+
+Examples:
+
+    jenkins-jobs --conf jenkins.ini update jjb / <job_name>
```
Comment (tykeal) [inline README.md:74]: that / is bad, also you need a -j after update

**Review by tykeal — CHANGES_REQUESTED** [review]
The check_* files. I don't like that. Those should get moved to global-jjb and have tox reference them from there. This is way too prone to file drift with them as standalone files

See the rest of my inlined comments

**Review by tykeal — APPROVED** [review]

**Pull request merged**